code/modules/mob/living/carbon/human/human.dm

```diff
@@ -949,10 +949,24 @@
 	return (ishuman(target) && !(target.mobility_flags & MOBILITY_STAND))
 
 /mob/living/carbon/human/proc/fireman_carry(mob/living/carbon/target)
+	var/carryspeed = 0 //if you have latex you are faster at grabbing
+	if(HAS_TRAIT(src, TRAIT_QUICKER_CARRY))
+		carryspeed = 35
+	else if(HAS_TRAIT(src, TRAIT_QUICK_CARRY))
+		carryspeed = 40
+	else 
+		carryspeed = 50

```

Comment:
don't need this, just set the default number

Reply:
did it